scripts/release/chebi_update/run_ChEBI_Update.pl

```diff
@@ -0,0 +1,75 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $data_release_pipeline_application_version = "0.0.1-SNAPSHOT";
+my $data_release_pipeline_application = "chebi-update";
+my $data_release_pipeline_tag = "";
+my $data_release_pipeline_repository = "https://github.com/reactome/data-release-pipeline";
```

Comment:
In the future I wonder if we should just clone a single repo somewhere and then have our (temporary) Perl scripts just access that single repo

Reply:
Yes, cloning it once would be best. We can figure that out before the next release.